react/week3/my_first_app/src/index.js

```diff
@@ -0,0 +1,255 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+//import App from './App';
+import Header from './components/Header';
+import Counter from './components/Counter';
+import * as serviceWorker from './serviceWorker';
+import "react-datepicker/dist/react-datepicker.css";
+
+const root = document.getElementById('root');
+
+function fetchTodo() {
+  const URL = `https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw`
+  return fetch(URL)
+    .then(response => response.json())
+}
+
+class TodoListItem extends React.Component {
+
+  render() {
+    const isEdit = this.props.todo.canEdit
+    const isUpdate = this.props.todo.canUpdate
+    const isComplete = this.props.todo.completed
+    const description = this.props.todo.description
+    const deadline = this.props.todo.deadline
+    const id = this.props.todo.id
+    const toggle = this.props.onToggle
+    const deleteList = this.props.onDelete
+    const editList = this.props.onEdit
+    const updateList = this.props.onUpdate
```

Comment:
It's not necessary to assign all of the props to variables like this. It's adequately clear to reference props directly. If you did want to make things more clear, consider destructuring instead
```suggestion
const { todo, onToggle, onDelete, onEdit, onUpdate } = this.props
```